postgres-connector/src/main/java/org/odpi/openmetadata/adapters/connectors/integration/postgres/PostgresDatabaseConnector.java

```diff
@@ -999,14 +1009,14 @@ private void addColumnsForTable(PostgresTable table, String tableGUID) throws Co
                 colProps.setDisplayName(col.getColumn_name());
                 colProps.setQualifiedName(col.getQualifiedName());
                 colProps.setAdditionalProperties(col.getProperties());
-
                 String colGUID = this.context.createDatabaseColumn(tableGUID, colProps);
 
```

Comment:
see the codeQL notice (String colGUID is never read)
Wonder if it would be useful to turn on enforcement - let me know if so. or can leave as is

Reply:
In this case the variable is being returned by the API , the connector isn't actually instantiating it , so a bit of a false positive.
I think we should definitely run linting though , can we turn on linting without it being compulsory ?

Reply:
I guess the approach there would be to explicitly cast the return to void - that way the code is clear on it's intent to never make use of the returned value?

For linting -- you can change the gradle build as you feel is suitable - and set compiler options. I've not tried -- but If you'd like me to take a look please raise an issue

Reply:
Nope your fine I can take it for now .